2996_reflection/reflection.md

```diff
@@ -271,4 +183,202 @@ std::string name1 = name_of(^S);             // Also okay.
 
 ## Metafunctions
 
+We propose a number of metafunctions declared in namespace `std::meta` to operator on reflection values.
+Adding metafunctions to an implementation is expected to be relatively "easy" compared to implementing the core language features described previously.
+However, despite offering a normal C++ function interface, each on of these relies on "compiler magic" to a significant extent.
```

Comment:
I think probably want to add a synopsis too. Just nice to have everything in one place.
```cpp
namespace std::meta {
    consteval auto invalid_reflection(string_view message, source_location loc = source_location::current()) -> info;
    consteval auto is_invalid(info) -> bool;
    // ...
}
```